website/docs/en/blog/announcing-1-0.mdx

```diff
@@ -0,0 +1,209 @@
+
+![Who is using](https://assets.rspack.dev/rspack/assets/rspack-v1-0-who-is-using.png)
+
+## How to use 1.0
+
+If you are using Rspack 0.7 or an earlier version, please note that 1.0 contains some breaking changes. We have prepared detailed documentation to help you upgrade. Please refer to: [Migration from Rspack 0.x](https://rspack.dev/guide/migration/rspack_0.x).
+
+If you have never used Rspack before, please see [Quick Start](https://rspack.dev/guide/start/quick-start) to get started with Rspack. Also, feel free to give a star 🌟 to the [Rspack GitHub repository](https://github.com/web-infra-dev/rspack).
+
+## What's next
+
+Rspack 1.0 marks a new beginning. Following this release, the Rspack team will focus on the following goals:
+
+- **Develop Rspack 1.x.** Rspack 1.x will iterate over 12 to 18 months, bringing more new features and improvements.
+- **Release Rsbuild 1.0.** It is based on Rspack 1.0 and supports [multi-environment builds](https://rsbuild.dev/guide/advanced/environments). Currently, Rsbuild has released version 1.0 RC, and the official release is expected in September.
+- **Release Rsdoctor 1.0.** This release will improve support for Vue and provide report formats for CI/CD.
+- **Develop Rslib 0.x.** Rslib is a library building tool based on Rsbuild. See [Rslib repository](https://github.com/web-infra-dev/rslib) for more details.
+- **Develop Rspress 2.0.** It will be based on React 19 and will improve some of the API designs. See [Rspress v2.0 planning](https://github.com/web-infra-dev/rspress/discussions/1105) for more details.
+
+Here are some key features we plan to support in Rspack 1.x:
+
+### Faster HMR
+
+Rspack can currently meet the performance requirements for most projects, but there is still significant room for performance optimization. During development, Rspack has already achieved nearly constant level incremental builds during the make phase. However, in the seal phase, some computations can still slow down as the project scales. Rspack will incrementally optimize the computations in the seal phase to keep the HMR time at a constant level.
+
+### Portable cache
+
+The evolution path of Rspack's caching capabilities follows a sequential implementation of memory cache, persistent cache, and portable cache. Currently, Rspack has implemented a memory cache that provides excellent HMR performance. The next step is to implement a persistent cache based on this foundation to address long cold startup times for large projects and to functionally align with webpack.
+
+After that, we plan to continue implementing the portable cache. This means that Rspack's build cache will not only be persistent, but also portable across environments and machines. This will help teams make better use of the cache and lay the groundwork for distributed builds.
```

Comment:
Is there any PR related to portable cache ? Didn't see it btw. 

Also using `the` means you are referring to a concept you have mentioned or at least people know it, but if no reference then we can make it bold (we are explaining it next line) ? 

```suggestion
After that, we plan to continue implementing **portable cache**. This means that Rspack's build cache will not only be persistent, but also portable across environments and machines. This will help teams make better use of the cache and lay the groundwork for distributed builds.
```

Reply:
Just planning, no PR yet